data/events.yaml

```diff
@@ -33,6 +33,13 @@
   location: Providence, RI, USA
   type: tutorial
 
+- title: Formalization of cohomology theories
+  url: https://www.birs.ca/events/2023/5-day-workshops/23w5124
+  start_date: May 21 2022
```

Comment:
The year should be 2023 (and presumably this should be somewhere higher up in the file to keep the chronological order)

Reply:
Should be fixed. Ordered the events by `end_date`. `start_date` would provide a different ordering.